source/manuals/programming-languages/nodejs/index.html.md.erb

```diff
@@ -6,8 +6,8 @@ title: Using Node.js at GDS
 
 ## Important Note
 
-No new projects at GDS should use Node.js without prior consultation with the 
-Deputy Director Technology Operations. The [programming language recommendations](../../../standards/programming-languages.html#javascript) 
+New projects at GDS should never use Node.js without prior consultation with the 
+GDS Way Forum. Topics like this are discussed at our monthly meetings. The [programming language recommendations](../../../standards/programming-languages.html#javascript) 
```

Comment:
How do you consult the GDS Way Forum?

Reply:
Good question... @pgchamberlin / @rboulton how do we consult the forum :)

Reply:
Email to `programme-technical-leaders@digital.cabinet-office.gov.uk` explaining the need and linking to whatever else might be relevant.

Reply:
Updated